extract/dcinside/refactor3.py

```
@@ -0,0 +1,541 @@
+                "car_id": airflow_json['car_id'],
+                "date": airflow_json['date'],
+                "batch": airflow_json['batch'],
+                "start_datetime": airflow_json['start_datetime'],
+                "end_datetime": airflow_json['start_datetime'],
+                "attempted_posts_count": tried,
+                "extracted_posts_count": tried - failed
+                }
+        }
+        pprint.pprint(return_params)
+    
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)